nomenclature/testing.py

```diff
@@ -24,3 +46,12 @@ def assert_valid_yaml(path):
         raise AssertionError(
             "Parsing the yaml files failed. Please check the log for details."
         )
+
+
+def check_valid_structure(path: Path):
+    """Check that "definition" folder in `path` exists and
+    can be initialized without errors"""
+    nomenclature.DataStructureDefinition(path / "definitions")
+
```

Comment:
I deleted the assertions since both of them get checked in the class `DataStructureDefinition` in the core.py. I hope this is fine like that.

Reply:
Should I also write a test for the function _assert_valid_structure_ in the test_testing.py or since core.py is tested in test_core.py it would be redundant?

Reply:
Agree, this is tested as part of `test_cli_valid_structure`, good enough for now.